build_tools/packaging/linux/packaging_utils.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import json
+
+def read_package_json_file():
+    '''Reads a JSON file and returns the parsed data.'''
+    with open("package.json", 'r') as file:
+        data = json.load(file)
+    return data
+
+
+def is_packaging_disabled(package):
+    '''
+    Checks if packaging is disabled for a given package.
+
+    Parameters:
+    package (dict): A dictionary containing package details.
+
+    Returns:
+    bool: True if 'DisablePackaging' key exists, False otherwise.
+    '''
+    return "DisablePackaging" in package
+
+
+def get_package_info(pkgname):
+    '''Function to retrieve package details stored in a JSON file for the provided package name'''
+    # Load JSON data from a file
+    data = read_package_json_file()
+
+    for package in data:
+        if package.get("Package") == pkgname:
+            return package
+            break
```

Comment:
No need for `break` after `return`
```suggestion
    for package in data:
        if package.get("Package") == pkgname:
            return package
```